web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/PopulateFiltersServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.rehabstod.service.sjukfall;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.rehabstod.auth.RehabstodUser;
+import se.inera.intyg.rehabstod.integration.it.dto.PopulateFiltersRequestDTO;
+import se.inera.intyg.rehabstod.integration.it.service.IntygstjanstRestIntegrationService;
+import se.inera.intyg.rehabstod.service.user.UserService;
+import se.inera.intyg.rehabstod.web.controller.api.dto.PopulateFiltersResponseDTO;
+import se.inera.intyg.rehabstod.web.controller.api.util.ControllerUtil;
+import se.inera.intyg.rehabstod.web.model.Lakare;
+
+@Service
+public class PopulateFiltersServiceImpl implements PopulateFiltersService {
+
+    private final UserService userService;
+    private final IntygstjanstRestIntegrationService intygstjanstRestIntegrationService;
+
+    public PopulateFiltersServiceImpl(UserService userService,
+        IntygstjanstRestIntegrationService intygstjanstRestIntegrationService) {
+        this.userService = userService;
+        this.intygstjanstRestIntegrationService = intygstjanstRestIntegrationService;
+    }
+
+    @Override
+    public PopulateFiltersResponseDTO get() {
+        final var user = userService.getUser();
+        final var unitId = ControllerUtil.getEnhetsIdForQueryingIntygstjansten(user);
```

Comment:
I think there is a missmatch here, should we change it so its the same as in GetActiveSickLeavesServiceImpl?

```java
 final var careUnitId = ControllerUtil.getEnhetsIdForQueryingIntygstjansten(user);
 final var unitId = user.isValdVardenhetMottagning() ? user.getValdVardenhet().getId() : null;
```